### ISSUE TITLE
Support full new schema

### ISSUE THREAD

**Issue by hugo-vrijswijk** [stryker-mutator/stryker4s#677]
The [latest mutation-testing-report-schema release](https://github.com/stryker-mutator/mutation-testing-elements/releases/tag/v1.5.1) added new properties to the schema. We should fill as many of them as we can:

- `testFiles` can be filled per `fullyQualifiedName` of a test suite
- `coveredBy` in a mutant
- `killedBy` in a mutant
- `duration` in a mutant
- `testsCompleted` in a mutant
- `performance`
- `framework`, we can fill `dependencies` with at least the sbt and stryker4s version. Perhaps the test framework too
- `system`, we can easily fill `ci`, `os` and `cpu`

**Comment by cbrunnkvist** [comment]
> https://raw.githubusercontent.com/stryker-mutator/mutation-testing-elements/master/packages/mutation-testing-report-schema/src/mutation-testing-report-schema.json

It seems like the mutation score per file is NOT included in the schema but, I presume, calculated on-the-fly by the UI component? 

That's problematic because I want to log the score to a custom graph after stryker4s is finished in CI. Isn't it also duplicating the score calculation in both `stryker4s.report.ConsoleReporter` _and_ the web UI (_and_ the Dashboard?). 🤔

Any ideas of how I can get the score out into my CI runner would be helpful - short of rendering the UI component in a virtual DOM...

**Comment by hugo-vrijswijk** [comment]
@cbrunnkvist  Yes, to prevent duplicate data in the JSON, the mutation score is calculated on the fly. 

Rendering the html report definitely doesn't seem ideal, haha. You might want to look at the [mutation-testing-elements](https://github.com/stryker-mutator/mutation-testing-elements) repo which has a library for [Scala](https://github.com/stryker-mutator/mutation-testing-elements/tree/master/packages/mutation-testing-metrics-scala) and [NodeJS](https://github.com/stryker-mutator/mutation-testing-elements/tree/master/packages/mutation-testing-metrics) to calculate metrics based on a report in a standard way. That code is used by StrykerJS, Stryker4s, the html report and the dashboard to calculate the score (and a couple of other things).

If there's something missing in the metrics, or it doesn't fit your use-case, let me know! We're happy to support more use-cases!
